src/validations/signup.js

```diff
@@ -0,0 +1,54 @@
+import Joi from '@hapi/joi';
+
+const signupValidations = (data) => {
+  const schema = Joi.object({
+    firstName: Joi.string()
+      .min(3)
+      .regex(/[a-zA-Z]/)
+      .required()
+      .messages({
+        'string.base': 'First name must be string',
+        'string.empty': 'Please fill in your first name',
+        'string.pattern.base': 'First name must contain letters only',
+        'string.min': 'First name must be atleast {#limit} characters long',
+        'any.required': 'First Name is required'
+      }),
+    lastName: Joi.string()
+      .min(3)
+      .regex(/[a-zA-Z]/)
+      .required()
+      .messages({
+        'string.base': 'Last name must be string',
+        'string.empty': 'Please fill in your last name',
+        'string.pattern.base': 'Last name must contain letters only',
+        'string.min': 'Last name must be atleast {#limit} characters long',
+        'any.required': 'Last Name is required'
+      }),
+    Email: Joi.string()
+      .min(8)
+      .regex(/[a-zA-Z]/)
+      .required()
+      .email()
+      .messages({
+        'string.base': 'Email name must be string',
+        'string.empty': 'Please fill in your email',
+        'string.min': 'email must be atleast {#limit} characters long',
+        'any.required': 'email is required'
+      }),
```

Comment:
These error messages should use the correct property name, i.e if the missing property is _Email_ it shouldn't send _email_ is required.